VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +77,86 @@ public void Operators()
         {
             int a = 0;
             int b = a + 13; 
-            Assert.IsTrue(b == /*Answer here*/); // Make this pass
+            Assert.IsTrue(b == 13); // Make this pass
 
             float f = 12.3f;
             float c = f - 7.43f;
-            Assert.IsTrue(c == /*Answer here*/); // Make this pass
-            Assert.IsTrue(f == /*Answer here*/); // Make this pass
+            Assert.IsTrue(c == 4.87f); // Make this pass
```

Comment:
This one also doesn't pass because of the way floating point numbers are stored.